esignet-default.properties

```diff
@@ -97,16 +125,15 @@ mosip.esignet.kafka.linked-auth-code.topic=esignet-consented
 
 ## ------------------------------------------- Integrations ------------------------------------------------------------
 
-mosip.esignet.integration.scan-base-package=io.mosip.esignet.mock.integration,io.mosip.authentication.esignet.integration
+mosip.esignet.integration.scan-base-package=io.mosip.authentication.esignet.integration,io.mosip.esignet.mock.integration
 mosip.esignet.integration.binding-validator=BindingValidatorServiceImpl
 mosip.esignet.integration.authenticator=IdaAuthenticatorImpl
 mosip.esignet.integration.key-binder=IdaKeyBinderImpl
-mosip.esignet.integration.audit-plugin=LoggerAuditService
+mosip.esignet.integration.audit-plugin=IdaAuditPluginImpl
```

Comment:
For now for testing lets switch to LoggerAuditService

Reply:
switched